okio/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply plugin: 'org.jetbrains.kotlin.platform.jvm'
+
+sourceCompatibility = JavaVersion.VERSION_1_7
+targetCompatibility = JavaVersion.VERSION_1_7
+
+dependencies {
+  implementation deps.kotlin.stdLib.jre7
```

Comment:
ditto Mike's comment: jre6?

Reply:
We should probably move everything to jre7 (which is actually jdk7 now)

Reply:
do we use anything from 7? i wouldn't bother unless we do

Reply:
Diamond operator, underscores in numeric literals

Reply:
That is not tied to this dependency